src/views/Game/GameDock.tsx

```diff
@@ -325,123 +329,157 @@ export function GameDock({
+                        user.anonymous ||
+                        engine.rengo ||
+                        (goban.isAnalysisDisabled() && phase !== "finished")
                             ? "disabled"
                             : ""
                     }
                 >
-                    <i className="fa fa-refresh"></i> {_("Review this game")}
+                    <i className="fa fa-code-fork"></i> {_("Fork game")}
                 </a>
-            )}
-            <a
-                onClick={onEstimateClicked}
-                className={phase !== "finished" && goban.isAnalysisDisabled() ? "disabled" : ""}
-            >
-                <i className="fa fa-tachometer"></i> {_("Estimate score")}
-            </a>
-            <a
-                onClick={fork}
-                className={
-                    user.anonymous ||
-                    engine.rengo ||
-                    (goban.isAnalysisDisabled() && phase !== "finished")
-                        ? "disabled"
-                        : ""
-                }
-            >
-                <i className="fa fa-code-fork"></i> {_("Fork game")}
-            </a>
-            <a onClick={alertModerator} className={user.anonymous ? "disabled" : ""}>
-                <i className="fa fa-exclamation-triangle"></i> {_("Call moderator")}
-            </a>
+            </Tooltip>
+            <Tooltip tooltipRequired={tooltipRequired} title={_("Call moderator")}>
+                <a onClick={alertModerator} className={user.anonymous ? "disabled" : ""}>
+                    <i className="fa fa-exclamation-triangle"></i> {_("Call moderator")}
+                </a>
+            </Tooltip>
             {((review && game_id) || null) && (
-                <Link to={`/game/${game_id}`}>
-                    <i className="ogs-goban" /> {_("Original game")}
-                </Link>
+                <Tooltip tooltipRequired={tooltipRequired} title={_("Original game")}>
+                    <Link to={`/game/${game_id}`}>
+                        <i className="ogs-goban" /> {_("Original game")}
+                    </Link>
+                </Tooltip>
             )}
-            <a onClick={showLinkModal}>
-                <i className="fa fa-share-alt"></i>{" "}
-                {review ? _("Link to review") : _("Link to game")}
-            </a>
-            {sgf_download_enabled ? (
-                <a href={sgf_url} target="_blank">
-                    <i className="fa fa-download"></i> {_("Download SGF")}
+            <Tooltip tooltipRequired={tooltipRequired} title={_("Link to game")}>
+                <a onClick={showLinkModal}>
+                    <i className="fa fa-share-alt"></i>{" "}
+                    {review ? _("Link to review") : _("Link to game")}
                 </a>
+            </Tooltip>
+            {sgf_download_enabled ? (
+                <Tooltip tooltipRequired={tooltipRequired} title={_("SGF Download")}>
```

Comment:
Let's use the same string here, `"Download SGF"` to avoid extra translation work (also to maintain consistency)

Reply:
Rewording to "Download SGF" causes an issue with one of the tests as two elements have the same string.  I am unsure of how to approach this?

![image](https://github.com/online-go/online-go.com/assets/76088738/a6ebc20b-24fa-48df-9ed6-b3b816aeff06)